src/obs-progress.cpp

```diff
@@ -71,14 +72,9 @@ void startTimer()
 
 void stopTimer()
 {
-	try
-	{
-		//_timer->stop();
-	}
-	catch (...)
-	{
-		// Empty
-	}
+	_timer->stop();
```

Comment:
Good catch! I removed the extra release.

Now I'm getting an exception here from Qt code.

```
qt5cored.dll!0x7ff930c35083
qt5cored.dll!0x7ff930c3559b
qt5cored.dll!0x7ff930c50856
obs-progress.dll!stopTimer+0x22
obs-progress.dll!obs_module_unload+0x1a
obs.dll!free_module+0x69
obs.dll!obs_shutdown+0x43e
obs64.exe!OBSApp::~OBSApp+0xa1
obs64.exe!run_program+0x86e
obs64.exe!main+0x6b8
obs64.exe!WinMain+0xed
obs64.exe!invoke_main+0x32
obs64.exe!__scrt_common_main_seh+0x12e
obs64.exe!__scrt_common_main+0xe
obs64.exe!WinMainCRTStartup+0x9
kernel32.dll!0x7ff9a20b6fd4
ntdll.dll!0x7ff9a2cdcec1
```